server/domains/shelter.py

```diff
@@ -0,0 +1,34 @@
+import uuid
+import dataclasses
+
+@dataclasses.dataclass
+class Shelter:
+    id: uuid.UUID
+    name: str
+    phone: str
+    city: str
+    state: str
+    zipCode: int
+    latitude: float
+    longitude: float
+    beds: int
+    distance: float
+    
+    @classmethod
+    def from_dict(self, d):
+        obj = {
+            "id": d["id"],
+            "name": d["name"],
+            "phone": d["phone"],
+            "city": d["city"],
+            "state": d["state"],
+            "zipCode": d["zipCode"],
+            "latitude": d["latitude"],
+            "longitude": d["longitude"],
+            "beds": d["beds"],
+            "distance": d["distance"],
+        }
+        return self(**obj)
+
+    def to_dict(self):
+        return dataclasses.asdict(self)
```

Comment:
I don't think we want to be duplicating shelter info in the volunteer app. The call to get a list of shelters can come directly from the front end. The user will then pick a specific shelter and the request to create a WorkShift coming to our server would only have some kind of unique shelter id. We could validate that this id is real by making a call to GetHelp's API, then create the WorkShift using that ID as the shelter.